client/public/assets/article-images/your-browser-support-toolkit.svg

```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 783 441" fill="none" role="img">
```

Comment:
This comment isn't about this exact `<svg ...` line but about the naming of the folder `article-images`. I think you should rename that to `deepdives`. I.e.
```
git mv client/public/assets/article-images client/public/assets/deepdives
```